### ISSUE TITLE
Chat input conflict with up/down arrows

### ISSUE THREAD

**Issue by reginacas** [reginacas/sense-ui#21]
While the functionality to use up and down arrows to access the command menu without having to use the / key was reported as good UX for some users, this is conflicting with a very important feature in NVDA, as reported by @samughal819 : 

> Problem: In NVDA, I use the Up and Down arrow keys to review the text I've typed in an edit field. In your extension, pressing the Up or Down arrow key at any time in the chat input opens the command suggestion list (/describe, /issues, etc.).
>     Impact: This makes it impossible for me to read or review my own message before sending it. It's very disorienting and was the source of my frustration. NVDA announces the input as a "combo box," which is confusing for a chat field.
>     Suggestion: The command list should only be triggered when the user types a / character, not by pressing the arrow keys. The input field should behave like a standard text area, allowing arrow keys to move the caret and review text.

Since this conflict is causing a major usability problem for the main use case of the extension, which is writing custom prompts, we will make sure the commands are only accessible when using the / key. 

https://github.com/reginacas/sense-ui/issues/8#issuecomment-3471105194
